Delete Asset - Oxmaint/readme.md

```diff
@@ -0,0 +1,32 @@
+## Delete Asset Oxmaint
+

```

Comment:
Please update readme.md file with all the required section, please refer below document. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md

Reply:
Updated